rust_dev_preview/cross_service/rest_ses/src/report.rs

```diff
@@ -92,7 +92,12 @@ pub async fn send_report(
 
     let email = ses
         .send_raw_email()
-        .raw_message(RawMessage::builder().data(data).build())
+        .raw_message(
+            RawMessage::builder()
+                .data(data)
+                .build()
+                .expect("building RawMessage"),
```

Comment:
we should probably make a conversion so you can do `?` here... 🤔

Reply:
Yeah, the new builders don't quite look as nice as I think you were going for.